src/lib/tooltip/README.md

```diff
@@ -1,24 +1,28 @@
 # MdTooltip
-Tooltip allows the user to specify text to be displayed when the mouse hover over an element.
+Tooltip allows the user to specify text to be displayed when the mouse hovers over an element.
+
+The positions `start` and `end` should be used instead of `left` and `right` respectively when the tooltip positioning should change depending on the HTML dir global attribute for left-to-right or right-to-left layout.
 
 ### Examples
 A button with a tooltip
 ```html
-<button md-tooltip="some message" tooltip-position="below">Button</button>
+<button md-tooltip="some message" tooltip-position="bottom">Button</button>
 ```
 
+
+
 ## `[md-tooltip]`
 ### Properties
 
 | Name | Type | Description |
 | --- | --- | --- |
 | `md-tooltip` | `string` | The message to be displayed. |
-| `tooltip-position` | `"above"|"below"|"before"|"after"` | The position of the tooltip. |
+| `tooltip-position` | `"start"|"end"|"top"|"bottom"|"left"|"right"` | The position of the tooltip. |
```

Comment:
Sorry, my brain latched on to `before` and `after` and I completely missed that we do want to keep `above` and `below` and relatively positioned items as well. 

(this is something we need to make sure is consistent before beta)